src/lib/services/universal-receiver.service.ts

```diff
@@ -84,12 +71,7 @@ export function universalReceiverDelegateDeploymentWithBaseContractAddress$(
 /**
  * TODO: docs
  */
```

Comment:
I think you can remove this todo, it is useless and also shows in the docs as a "TODO"

Reply:
I actually make a separate PR to update this docs so we can remove the TODOs